fhir-server/src/main/java/com/ibm/fhir/server/registry/ServerRegistryResourceProvider.java

```diff
@@ -225,21 +209,4 @@ public void afterDelete(FHIRPersistenceEvent event) {
 
         return Collections.emptyList();
     }
-
-    private void updateRegistryResourceMap(FHIRPersistenceEvent event) {
```

Comment:
I still think this was rather clever.  I'm going to try remembering this one and, maybe, reinstate it if/when we have a decentralized / external cache

Reply:
Agree it was clever. Will definitely consider bringing it back when we move to a distributed cache.